bluebottle/payments_flutterwave/models.py

```diff
@@ -0,0 +1,107 @@
+from decimal import Decimal
+import json
+
+from django.db import models
+from django.utils.translation import ugettext as _
+
+from bluebottle.payments.models import Payment
+
+
+class FlutterwavePayment(Payment):
+
+    amount = models.CharField(
+        help_text="Amount",
+        null=True, blank=True,
+        max_length=200)
+    currency = models.CharField(
+        help_text="Transaction currency",
+        default="NGN",
+        null=True, blank=True,
+        max_length=200)
+    auth_model = models.CharField(
+        help_text="Authentication Model - BVN, PIN, NOAUTH, VBVSECURECODE",
+        default="BVN",
+        null=True, blank=True,
+        max_length=200)
+    card_number = models.CharField(
+        help_text="Card Number",
+        null=True, blank=True,
+        max_length=200)
+    cvv = models.CharField(
```

Comment:
I haven't gone thru the whole api payment flow and dont know if the request to the api payment gateway is made synchronously during the request or asynchronously. I will go thru whole flow and also check up if there is any way of doing a one way hash of the information.